nano/node/bootstrap_ascending/peer_scoring.cpp

```diff
@@ -109,9 +113,24 @@ void nano::bootstrap_ascending::peer_scoring::sync (std::deque<std::shared_ptr<n
 			{
 				if (!channel->max (nano::transport::traffic_type::bootstrap))
 				{
-					index.insert ({ channel, channel.get (), channel->get_endpoint (), 1, 1, 0 });
+					index.emplace (channel, 1, 1, 0);
```

Comment:
The counters could go also as default values in the constructor, not sure which way is better.